aiogram/types/message.py

```diff
@@ -1723,16 +1723,15 @@ def get_url(self, force_private: bool = False) -> str:
         :param force_private: if set, a private URL is returned even for a public chat
         :return: string with full message URL
         """
-        if self.chat.type == "private":
+        if self.chat.type in ("private", "group"):
```

Comment:
Why don't you use constants here?
`ChatType.PRIVATE` and `ChatType.GROUP`

Reply:
There is no ChatType class (_yet_)